docs/organizer-guide/newsletter/newsletter-process.rst

```diff
@@ -0,0 +1,91 @@
+#########################
+Write the Docs Newsletter
+#########################
+
+At the beginning of every month, we put out an email newsletter to the community. It's built around a handful of stories that we capture and write-up from the Slack team, and also includes an overview of what the org team is workingon and a calendar listing of upcoming events.
```

Comment:
`workingon` -- also could use sentence line breaks to make this review easier :D

Reply:
:smiling_imp:

Reply:
🙄